src/Uno.UI/UI/Xaml/FrameworkElement.iOS.cs

```diff
@@ -64,8 +64,6 @@ public override void LayoutSubviews()
 						var parent = Superview;
 						if (parent is UIElement
 						    || parent is ISetLayoutSlots
-						    // In the case of ListViewItem inside native list, its parent's parent is ListViewBaseInternalContainer
-						    || parent?.Superview is ISetLayoutSlots
```

Comment:
Very nice! Let's find out why this was originally needed.

Reply:
definitively not what I expected 😄

Reply:
@ajpinedam this was added in https://github.com/unoplatform/uno/pull/7399/files#diff-24c92c6bd56b6e501b90b4d9ccc6a72ea89c32b20b2992241cc7f072c0c42ddb. Would you know if the `When_ItemsPresenter_MinHeight` test is passing?